src/HTTPSock.cpp

```diff
@@ -181,6 +188,12 @@ void CHTTPSock::ReadLine(const CString& sData) {
             // X-Forwarded-For list in both cases use it as the endpoind
             m_sForwardedIP = sIP;
         }
+    } else if (sName.Equals("X-Forwarded-CertFP:")) {
+        bool bTrusted = IsTrustedProxy(GetRemoteIP());
+        if(bTrusted) {
```

Comment:
space missing
you can use clang-format to fix style with a single keypress, and not care about indentation and other things manually